pageserver/src/tenant.rs

```diff
@@ -3317,9 +3319,6 @@ impl TenantShard {
         match err {
             err if err.is_cancel() => {}
             CompactionError::ShuttingDown => (),
-            // Offload failures don't trip the circuit breaker, since they're cheap to retry and
-            // shouldn't block compaction.
-            CompactionError::Offload(_) => {}
```

Comment:
What about this one? offload failures now trip the circuit breaker.

Reply:
No, as pointed out in the PR description, `maybe_trip_compaction_breaker` is never called for errors from the `offload_timeline` call